src/compas_fab/robots/robot.py

```diff
@@ -799,97 +885,137 @@ def orientation_constraint_from_frame(self, frame_WCF, tolerances_axes,
+        group: :class:`str`, optional
             The planning group for which we specify the constraint. Defaults to
             the robot's main planning group.
 
+        Returns
+        -------
+        :class:`list` of :class:`Constraint`
+
+        See Also
+        --------
+        * :meth:`PositionConstraint.from_box`
+        * :meth:`PositionConstraint.from_mesh`
+        * :meth:`PositionConstraint.from_sphere`
+        * :meth:`orientation_constraint_from_frame`
+
+        Notes
+        -----
+        The rotation tolerance for an axis is defined by the other vector
+        component values for rotation around corresponding axis.
+        If you specify the tolerances_axes vector with ``[0.01, 0.01, 6.3]``, it
+        means that the frame's x-axis and y-axis are allowed to rotate about the
+        z-axis by an angle of 6.3 radians, whereas the z-axis would only rotate
+        by 0.01.
+
         Examples
         --------
         >>> frame = Frame([0.4, 0.3, 0.4], [0, 1, 0], [0, 0, 1])
         >>> tolerance_position = 0.001
         >>> tolerances_axes = [math.radians(1)]
         >>> group = robot.main_group_name
         >>> robot.constraints_from_frame(frame, tolerance_position, tolerances_axes, group)
-        [PositionConstraint('ee_link', BoundingVolume(2, Sphere(Point(0.400, 0.300, 0.400), 0.001)), 1.0), \
-        OrientationConstraint('ee_link', [0.5, 0.5, 0.5, 0.5], [0.017453292519943295, 0.017453292519943295, 0.017453292519943295], 1.0)]
-
-        Notes
-        -----
-        There are many other possibilities of how to create a position and
-        orientation constraint. Checkout :class:`compas_fab.robots.PositionConstraint`
-        and :class:`compas_fab.robots.OrientationConstraint`.
-
-        """
+        [PositionConstraint('ee_link', BoundingVolume(2, Sphere(Point(0.400, 0.300, 0.400), 0.001)), 1.0), OrientationConstraint('ee_link', [0.5, 0.5, 0.5, 0.5], [0.017453292519943295, 0.017453292519943295, 0.017453292519943295], 1.0)]
+        """  # noqa: E501
         pc = self.position_constraint_from_frame(frame_WCF, tolerance_position, group)
         oc = self.orientation_constraint_from_frame(frame_WCF, tolerances_axes, group)
         return [pc, oc]
 
     def constraints_from_configuration(self, configuration, tolerances_above, tolerances_below, group=None):
-        """Returns joint constraints on all joints of the configuration.
+        """Create joint constraints for all joints of the configuration.
 
         Parameters
         ----------
-        configuration: :class:`compas_fab.robots.Configuration`
+        configuration: :class:`Configuration`
             The target configuration.
-        tolerances_above: list of float
+        tolerances_above: :class:`list` of :class:`float`
             The tolerances above the targeted configuration's joint value on each
             of the joints, defining the upper bound in radian to be achieved.
```

Comment:
Here and below I think it should read "in radians".

Reply:
7777ace